src/Main.java

```diff
@@ -1,5 +1,93 @@
+import java.util.Random;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+//        // task 1
+        System.out.println(findSymbolOccurance(null, 'a'));
+//        // task 2
+        System.out.println(findWordPosition("Apollo", "pollo"));
+        System.out.println(findWordPosition("Apple", "Planet"));
+//        // task 3
+        System.out.println(stringReverse("Hello"));
+        System.out.println(stringReverse2("Java"));
+//        // task 4
+        System.out.println(isPalindrome("ERE"));
+        System.out.println(isPalindrome("Java"));
+        // task 5
+        System.out.println(wordGame());
+
+    }
+    public static int findSymbolOccurance(String word, char letter) {
+        int counter = 0;
+        if (word == null) {
```

Comment:
Would be better to validate an argument first, and then declare `counter` variable.